tools/cythonize.py

```diff
@@ -76,8 +76,11 @@ def process_pyx(fromfile, tofile):
         # versions as they become required for new python versions.
         if sys.version_info[:2] < (3, 7):
             required_version = LooseVersion('0.19')
-        else:
+        elif sys.version_info[:2] < (3, 8):
```

Comment:
This isn't right, we definitely have required fixes in 0.29.X independent of Python version. Let's just remove this whole switching logic and keep it as `>= 0.29.13` for all supported Python versions.

Reply:
Heh, that was my first thought too. Will do.